client/login/wp-login/index.jsx

```diff
@@ -70,10 +70,11 @@ export class Login extends React.Component {
 	}
 
 	render() {
-		const { translate, twoFactorAuthType } = this.props;
+		const { translate, twoFactorAuthType, clientId } = this.props;
+		const title = clientId ? translate( 'Howdy! Log in to ' + clientId + ' with your WordPress.com account.' ) : translate( 'Log in to your account.' );
```

Comment:
Mentioning the app name on this page is a good thing for new users who click "login" in one site, then find themselves at a different, unfamiliar web site (WordPress.com.) It's also good for scenarios where the app forwarded the user to the login page abruptly (when a user's session has expired, and they follow a bookmark to an authentication-required page, for example.)

This is less of an issue for sites that replace "login" links with something more descriptive like "Sign In With WordPress.com". But I still think having the app name on this page will help people unfamiliar with WordPress.com understand what is going on.

Just my two cents. :)